static/styles/drafts.css

```diff
@@ -69,12 +69,14 @@
             color: hsl(0, 0%, 67%);
             pointer-events: none;
         }
+
+        h2 {
+            font-size: 1.1em;
+        }
     }
 }
 
 .draft-row {
-    padding: 5px 25px;
```

Comment:
Is this change related to the 25px in `drafts-list` above? It's possible that could be a prep CSS refactor.